src/main/java/bio/terra/app/configuration/DrsServiceConfiguration.java

```diff
@@ -0,0 +1,21 @@
+package bio.terra.app.configuration;
```

Comment:
I'm a little torn about where the RASIssuer setting should live.  

We already have a [drs setting](https://github.com/DataBiosphere/jade-data-repo/blob/develop/src/main/resources/application.properties#L59) in the [main configuration file](https://github.com/DataBiosphere/jade-data-repo/blob/develop/src/main/java/bio/terra/service/configuration/ConfigurationService.java). 

And, actually, this "issuer" refers to passports, so it might actually make sense to live in the ECMConfiguration. 
Thoughts?

Reply:
Good points.

The existing drs setting in the main config file strikes me as a holdout from an earlier time where all configs were grouped together, and not necessarily the best place for it.  I was reluctant to add on to that file.

But I agree that this fits in well with ECMConfiguration (after talking over Slack about what ECM is :) ).  I'll move it there.

Reply:
Moved this setting to ECMConfiguration and removed DrsServiceConfiguration.